index.bs

```diff
@@ -659,31 +805,43 @@ urlPrefix: https://w3c.github.io/FileAPI/#dfn-; type: dfn;
 		<h4 method for="Clipboard">readText()</h4>
 		The {{Clipboard/readText()}} method must run these steps:
 
-			1. Let |p| be a new [=Promise=].
+			1. Let |realm| be [=this=]'s [=relevant realm=].
+
+			1. Let |p| be [=a new promise=] in |realm|.
 
 			1. Run the following steps [=in parallel=]:
 
-				1. Let |r| be the result of running [=check clipboard read permission=] [=in parallel=]
+				1. Let |r| be the result of running [=check clipboard read permission=].
+
+					Note: Clipboard permission is not supported on Safari. However, the read() method must be called inside
```

Comment:
Copy-paste error: "the read() method" -> "the readText() method". In the same paragraph "read()" is mentioned again. this needs to be corrected. The same applies to "write" and "writeText".

Reply:
Oops, thanks for noticing this!